src/common/error.ts

```diff
@@ -29,6 +29,35 @@ export class UploadError extends Error {
   }
 }
 
+export class NoTokenListed extends Error {
```

Comment:
This shouldn't be its own error class. Make this an `invariant` call in the `list` method instead.